src/pages/learn/[id].tsx

```diff
@@ -148,9 +149,14 @@ function LearnDetail({ videoData, highlightData }: { videoData: VideoData; highl
                         onClick={() => player?.seekTo(startTime, true)}
                         isActive={startTime <= currentTime && currentTime < endTime ? true : false}>
                         <p id={id.toString()}>{text}</p>
-                        {clickedScriptId === id && !isNewMemo && (
-                          <ContextMenu points={points} setIsNewMemo={setIsNewMemo} />
-                        )}
+                        {clickedHighlightId === id && !memoHighlightId[0] && !memoHighlightId[1] ? (
+                          <ContextMenu
+                            points={points}
+                            hasMemo={hasMemo}
+                            id={id}
+                            setMemoHighlightId={setMemoHighlightId}
+                          />
+                        ) : null}
```

Comment:
```suggestion
                        {clickedHighlightId === id && !memoHighlightId[0] && !memoHighlightId[1] && (
                          <ContextMenu
                            points={points}
                            hasMemo={hasMemo}
                            id={id}
                            setMemoHighlightId={setMemoHighlightId}
                          />
                        )}
```

null을 넣어주는 대신 이렇게 처리하는 것은 어떨까요?